.changeset/orange-turtles-taste.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': minor
+'@firebase/auth': minor
+---
+
+Adding Persistence.COOKIE for use in frameworks that utilize hybrid rendering
```

Comment:
Could you punch-up this changeset description? It becomes our public release note.